docs/architecture.md

```diff
@@ -75,3 +116,146 @@ the decision.
 ### Attestation diagram
 
 ![attestation](images/Fobnail-flows-attestation.png)
```

Comment:
The Fobnail diagrams are unreadable in GitHub's dark mode. Please change the transparent background to white. Also applies to other.

Reply:
Should be fixed now

Reply:
@krystian-hebel Still `Fobnail-flows-fobnail-provisioning.png` and `Fobnail-flows-remote-platform-provisioning.png` have the transparent background.